Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:8.6-slim
+
+RUN apt-get update && apt-get -y install redis-server
```

Comment:
Redis should be a separate docker image.

Reply:
These two should be in the same Pod then?

Reply:
No. Two separate pods.

On your local machine, you will need to start up a Redis docker image.
In production, we will connect to a hosted Redis solution.

We don't want to run databases, cache-servers, or any other image that has permanent storage in the kubernetes cluster in production. We will rely on managed, hosted solutions.

But, for local testing we need to start up a seperate redis image to simulate the hosted service.